engine/jsonlint/pipe.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://repometric.github.io/linterhub/schema/extr.json",
+    "id": "jsonlint",
+    "name": "jsonlint",
+    "pipeline": [
+        {
+            "cmd": "jsonlint",
+            "success": 1,
+            "engine": true,
+            "args":{
+                "":"--compact",
```

Comment:
Why compact mode is used?

Reply:
it's output format of engine

Reply:
If we use some argument of linter in pipe file, we also remove `id` properties for this argument in `args.json` file.
`--comapct` argument need for convert output format of linter to our format.